sklearn/datasets/base.py

```diff
@@ -242,6 +242,122 @@ def load_files(container_path, description=None, categories=None,
                  DESCR=description)
 
 
+def load_data(module_path, data_file_name):
+    """
```

Comment:
I'm a little confused. I wrote a summary. Do you mean that my summary that I have written does not follow PEP257?